.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-  "jsxBracketSameLine": true,
+  "bracketSameLine": true,
```

Comment:
`jsxBracketSameLine` was [deprecated](https://prettier.io/docs/en/options.html#deprecated-jsx-brackets) in `prettier@2.4.0` in favor of `bracketSameLine`.